docs/development.rst

```diff
@@ -52,23 +52,19 @@ Quickstart
 Continuous integration tests
 ****************************
 
-Three different continuous integration providers are used, which run different
+Two different continuous integration providers are used, which run different
 combinations of tests on different platforms:
 
-1. A `Github action <https://help.github.com/en/actions>`_ runs `pre-commit
-   <https://pre-commit.com/>`_ to run a variety of code style and quality checks.
+1. `Github actions <https://help.github.com/en/actions>`_ run `pre-commit
+   <https://pre-commit.com/>`_ run a variety of code style and quality checks along
```

Comment:
This isn't quite parsing as english "run pre-commit run"

Reply:
Forest gump wrote that bit. Fixed!

Reply:
Haha, good one!